src/main/scala/mesosphere/marathon/DebugConf.scala

```diff
@@ -11,26 +11,22 @@ import org.rogach.scallop.ScallopConf
 
 /**
   * Options related to debugging marathon.
-  * All options should be optional and turned off by default.
   */
 trait DebugConf extends ScallopConf {
 
-  lazy val debugTracing = opt[Boolean]("enable_tracing",
-    descr = "Enable trace logging of service method calls",
+  lazy val debugTracing = toggle("tracing",
+    descrYes = "Enable trace logging of service method calls",
+    descrNo = "(Default) Disable trace logging of service method calls",
     default = Some(false),
-    noshort = true)
+    noshort = true,
+    prefix = "disable_")
 
-  @deprecated("Metrics are enabled by default. See disableMetrics.", "Since version 0.10.0")
-  lazy val enableMetrics = opt[Boolean]("enable_metrics",
```

Comment:
keep this as a hidden option to ignore the comment and allow Marathon to startup